Add responseSentTime to response format

berzniz opened the pull request:
Added `responseSentTime` to the response format and updated the unit tests. Questions left open:

1. This could be a breaking change for people who parse the logs. Should this be opt-in?
2. Should we tell which time is which in the log format?

Ideally, it should not be a breaking change and we will be able to pass in a format string for responses. Is this commit a good start?

Addresses https://github.com/hapijs/good-console/issues/94

arb commented:
1. I feel like if you are parsing strings... you'd have your own logger at this point. So I'd say just leave it as is.
2. Yes, put some really short copy in front of each value so you can tell which is which.

I'd just make this a semver major to avoid any potential breakages.

berzniz commented:
1. 👍 
2. I'll add the short copy, but what do you think should it be?
3. Tests are not passing which is weird, the error is `The following leaks were detected:WebAssembly`, any idea?

arb commented:
1. It's your PR, figured you'd have an idea what the short copy should be
2. While you're in there, try updating some of the development dependencies.

berzniz commented:
I added the label in front of the response times and updated the dev dependencies (+fixed one linting error)

Travis still fails, I think we need to run it without cache so new dependencies get installed. I can't configure it to clear the cache

berzniz commented:
Any updates on this? Are there any changes required?

arb commented:
Travis is still failing. I'm not sure what to do about it but this PR won't land until we get a clean run.

arb commented:
If you have any interest in landing this, please fix merge conflicts and get CI passing, if not, please close the PR.

berzniz commented:
Fixed the merge conflicts and the CI is passing

berzniz commented:
Close  #94
